types/artifacts.d.ts

```diff
@@ -287,6 +289,16 @@ declare module Artifacts {
 
   interface PasswordInputsWithPreventedPaste {node: NodeDetails}
 
+  interface Script extends Omit<LH.Crdp.Debugger.ScriptParsedEvent, 'url'|'embedderName'> {
```

Comment:
Do we need everything from `ScriptParsedEvent`? Imo it would be better to explicitly list the properties we want here and add them individually. That way we aren't including a bunch of stuff in the artifacts JSON that isn't needed.

Reply:
Is there a reason to give any thought to the size of artifacts?

Reply:
Ya I guess size of artifacts doesn't matter.

Side-note: Why do we need to omit `url` if we just re-declare it with the same type? Were you planning to modify the JSDOC comment or something?

Reply:
It indeed removes the jsdoc as defined in the protocol, and replaces it with nothing (since I didn't define a jsdoc here). interesting!

It was really just to make it clear I'm rewriting the field on purpose.